src/events/reactionAdded.js

```diff
@@ -58,7 +58,7 @@ export default async ({ event }) => {
   if (
     reaction === "scrappy-retry" &&
     channel == process.env.CHANNEL &&
-    message
+    message && !message.thread_ts
```

Comment:
Will this have unintended behavior if there's already a thread on a message when :scrappy-retry: is reacted? (I'm thinking in non-#scrapbook channels where people might scrapbook.

Reply:
This should not have such effect! You're welcome to test this though